src/Slider/TimeSlider/TimeSlider.spec.tsx

```diff
@@ -9,102 +11,17 @@ describe('<TimeSlider />', () => {
   });
 
   it('can be rendered', () => {
-    const wrapper = TestUtil.mountComponent(TimeSlider, {});
```

Comment:
did you intent to remove most of the tests? I know they're not testing-library tests, but they seem to be still useful, aren't they?

Reply:
maybe you could a few more advanced testing-library tests, if you have the time

Reply:
Sure, I can look for some testing-library tests. I have removed theses tests, because after turning the component into a functional component, `instance()` returned `null` as this [isn't supported anymore](https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/instance.html). It is also recommended to test the behavior of the component, not the implementation details.

Reply:
I just mean going beyond the 'component can be rendered'. In this case it could mean providing some layers and a start and end date. As it is done in the file TimeLayerSiderPanel.example.md

There are many components where we have useful testing-library tests, for instance https://github.com/terrestris/react-geo/blob/rg-future/src/Field/ScaleCombo/ScaleCombo.spec.tsx

Reply:
agree about removing the tests though. We don't need to replace them 1:1